docs/testing.md

```diff
@@ -320,8 +320,7 @@ automatically in Zulip's Travis CI tests to avoid any regressions, but
 is not included in test-all since it is quite slow.
 
 To run tooks/check-py3, you need to install the modernize and future
-python packages (which are in the development environment's
-requirements.txt file).
+python packages (which are in requirements/dev.txt).
```

Comment:
Yeah, or maybe just change the parenthetical (which are in the development environment's requirements.txt file, so you probably already have them installed).